sal_interfaces/CCCamera/CCCamera_Commands.xml

```diff
@@ -0,0 +1,292 @@
+        <IDL_Type>boolean</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>wfs</EFDB_Name>
+        <Description>True if wavefront sensors should be read out</Description>
+        <IDL_Type>boolean</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>timeout</EFDB_Name>
+        <Description>Timeout after which imaging will be abandoned if no endImage command received</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>second</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>imageType</EFDB_Name>
+        <Description>Image type (a.k.a. IMGTYPE) (e.g. e.g. BIAS, DARK, FLAT, FE55, XTALK, CCOB, SPOT...)</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>256</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>groupId</EFDB_Name>
+        <Description>Image groupId. Used to fill in FITS GROUPID header</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>256</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALCommand>
+<SALCommand>
+    <Subsystem>CCCamera</Subsystem>
+    <Version>3.7.1</Version>
+    <Author></Author>
+    <EFDB_Topic>CCCamera_command_disableCalibration</EFDB_Topic>
+    <Alias>disableCalibration</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>Unused</Description>
+        <IDL_Type>boolean</IDL_Type>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALCommand>
+<SALCommand>
+    <Subsystem>CCCamera</Subsystem>
+    <Version>3.7.1</Version>
+    <Author></Author>
+    <EFDB_Topic>CCCamera_command_initGuiders</EFDB_Topic>
+    <Alias>initGuiders</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>roiSpec</EFDB_Name>
+        <Description>TBD specification for region of interest</Description>
+        <IDL_Type>string</IDL_Type>
+        <IDL_Size>256</IDL_Size>
+        <Units>unitless</Units>
+        <Count>1</Count>
+    </item>
+</SALCommand>
+<SALCommand>
+    <Subsystem>CCCamera</Subsystem>
+    <Version>3.7.1</Version>
+    <Author></Author>
+    <EFDB_Topic>CCCamera_command_enableCalibration</EFDB_Topic>
+    <Alias>enableCalibration</Alias>
+    <Device></Device>
+    <Property></Property>
+    <Action></Action>
+    <Value></Value>
+    <Explanation>http://project.lsst.org/ts/sal_objects/help</Explanation>
+    <item>
+        <EFDB_Name>value</EFDB_Name>
+        <Description>Unused</Description>
```

Comment:
The description for unused parameters is: _Attribute required by the API, but is unused_.

Reply:
This is fixed.